src/shared/Card/Card.js

```diff
@@ -6,36 +6,46 @@ import Badges from '../Badges';
 
 import './Card.css';
 
-const Card = ({ title, link, badges = {}, description, products = {}, services = {} }) => (
-  <a href={link}>
-    <section className="card" styleName="card">
-      <article>
-        <h3>
-          <span>{title}</span>
-        </h3>
-        {!isEmpty(badges) && (
-          <div styleName="badges">
-            <Badges badges={badges} />
-          </div>
-        )}
-        <p styleName="description">{description}</p>
-        {/* TODO refactor when products exists
-        <div className="products">
-          <Icon value="product" size={18} /> <strong>Digital products:</strong> CMS
-        </div>
-        */}
-      </article>
-
-      {!isEmpty(services) && (
-        <div styleName="services">
-          {Object.keys(services).map((service, i) => (
-            <span key={i}>{service}</span>
-          ))}
-        </div>
-      )}
-    </section>
-  </a>
-);
+const Card = ({ title, link, badges = {}, description, products = {}, services = {} }) => {
+    // calculate badges
+    var calculated_badges = new Object();
+    Object.assign(calculated_badges, badges);
+    //calculated_badges['recruitment'] = (recruiter === 'yes');
+    calculated_badges['product'] = !isEmpty(products);
+    badges = calculated_badges;
```

Comment:
not a fan of mutating props, perhaps better to:
`badges = Object.assign({}, product: !isEmpty(products), badges);`